express-notes-api/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require('express');
+const dataJSON = require('./data.json');
+const fs = require('fs');
+
+const app = express();
+const middlewareJSON = express.json();
+
+app.get('/api/notes', (req, res) => {
+  const notesArray = [];
+  for (const property in dataJSON.notes) {
+    notesArray.push(dataJSON.notes[property]);
+  }
+  res.status(200).json(notesArray);
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  const id = req.params.id;
```

Comment:
I would use the `Number` constructor here, to convert the string value to a number